docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -150,81 +150,81 @@ values requiring extrapolation being masked.
 
 .. _regridding:
 
-Regridding
----------------------------------
 
-Regridding conceptually is a very similar to interpolation in Iris, with the primary difference being
-that interpolations are based on sample points, where regridding is based on the **spatial** grid of
-*another cube*.
+Regridding
+----------
 
-Regridding is achieved with the :meth:`cube.regrid() <iris.cube.Cube.regrid>` method,
-with the first argument being *another cube* which has the grid to which the cube should
-be interpolated onto, and the second argument being the regridding scheme to use.
+Regridding is conceptually a very similar process to interpolation in Iris. 
+The primary difference is that interpolation is based on sample points, while
+regridding is based on the **spatial** grid of *another cube*.
 
-The current regridding schemes available are :class:`iris.analysis.Linear` for a linear point
-based regrid and :class:`iris.analysis.AreaWeighted` for area weighted regridding.
+Regridding a cube is achieved with the :meth:`cube.regrid() <iris.cube.Cube.regrid>` method.
+This method expects two arguments; the first argument being *another cube* that defines
+the grid onto which the cube should be regridded, and the second argument being
+the regridding scheme to use.
 
 .. note::
 
-    Regridding is a common operation needed to allow comparisons of data on different grids, however
-    because of the powerful mapping functionality provided by cartopy, regridding is often not
-    necessary if it is just for visualisation purposes.
+    Regridding is a common operation needed to allow comparisons of data on different grids.
+    The powerful mapping functionality provided by cartopy, however, means that regridding
+    is often not necessary if performed just for visualisation purposes.
 
-Let's load two cubes which are on different grids:
+Let's load two cubes that are on different grids:
 
     >>> global_air_temp = iris.load_cube(iris.sample_data_path('air_temp.pp'))
     >>> rotated_psl = iris.load_cube(iris.sample_data_path('rotated_pole.nc'))
 
-We can visually confirm that they are on different grids by drawing a block plot
-(pcolormesh) of the two cubes:
+We can visually confirm that they are on different grids by plotting the two cubes:
 
 .. plot:: userguide/regridding_plots/regridding_plot.py
 
-To regrid the air temperature values onto the rotated pole grid using a linear
-interpolation scheme, we pass the ``rotated_psl`` cube, whose grid will be used
-as the locations for the interpolated air temperature values:
+Let's regrid the ``global_air_temp`` cube onto a rotated pole grid
```

Comment:
should we clarify that we're going from geographic lat/lon, since we're being explicit about the coordinate-system of the target?